### DISCUSSION TITLE
Unattended Install of Mirth 4.3?

### DISCUSSION THREAD

**Discussion by mcswaip1** [nextgenhealthcare/connect#5769] [Q&A] [answered]
With all the changes that were done to the Mirth 4.3 installer, is it possible to run the installer in an unattended mode providing the installation settings on the command line?
I know that we can run it with just `-q`, but that doesn't set any of the values for the install and it still seems to try running the Administrator Launcher Installer ... so not so quiet...
Thanks
Paul.

**Comment by ab-20-m** [comment, marked as answer]
Try `-q -varfile response.var`, where response.var is a response file with your desired parameters. When you do a full UI install the options/inputs you entered in the UI are saved under `.install4j\response.varfile` relative to your Mirth install directory.
